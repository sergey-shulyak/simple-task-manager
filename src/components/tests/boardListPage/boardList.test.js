import React from 'react';
import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';

import BoardList from '../../boardListPage/boardList';
import BoardEntry from '../../boardListPage/boardList/boardEntry';

const children = [
  <BoardEntry key="1" id="1" title="test" />,
  <BoardEntry key="2" id="2" title="test" />,
  <BoardEntry key="3" id="3" title="test" />
];

describe('BoardList', () => {
  it('Should render correctly', () => {
    const wrapper = shallow(<BoardList>{children}</BoardList>);

    expect(toJson(wrapper)).toMatchSnapshot();
  });
});
