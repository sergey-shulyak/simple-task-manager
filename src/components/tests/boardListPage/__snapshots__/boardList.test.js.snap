// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`BoardList Should render correctly 1`] = `
<div
  className="board-list "
>
  <BoardEntry
    className=""
    description=""
    id="1"
    key="1"
    showDeleteModal={[Function]}
    title="test"
  />
  <BoardEntry
    className=""
    description=""
    id="2"
    key="2"
    showDeleteModal={[Function]}
    title="test"
  />
  <BoardEntry
    className=""
    description=""
    id="3"
    key="3"
    showDeleteModal={[Function]}
    title="test"
  />
</div>
`;
